tests/FileTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * (c) 2015 Rebilly SRL
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Rebilly\Tests;
+
+use Rebilly\ApiKeyProvider;
+use Rebilly\Client;
+use Rebilly\Rest\File;
+use RuntimeException;
+
+/**
+ * Class FileTest.
+ *
+ * @group wip
+ */
+class FileTest extends TestCase
+{
+    public function testFile()
+    {
+        $client = new Client(['apiKey' => 'QWERTY']);
+
+        $client = new Client(
+            [
```

Comment:
It is also a controversial issue. So in [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#44-method-arguments) written:

> Argument lists MAY be split across multiple lines, where each subsequent line is indented once. When doing so, the first item in the list MUST be on the next line, and there MUST be only one argument per line.

Seems this variant more correct, then oldest. I never really thought about it.